### DISCUSSION TITLE
Restart footnotes each chapter

### DISCUSSION THREAD

**Discussion by mk-uc** [jgm/pandoc#9999] [Q&A]
I'm wondering if there's any way within Pandoc to restart the footnotes for each chapter when converting to docx. I can go in to Word afterward and do this—but this doesn't entirely solve the problem because references that have already been cited use their short form, and I think they need to be in their long, full reference.

**Comment by tarleb** [comment]
I'm not sure if I fully understand. Do those footnotes come from a citations? If so, maybe the [section-bibliographies](https://github.com/pandoc-ext/section-bibliographies) filter can help.

**Comment by denismaier** [comment]
I think the question is back references in citations. Something like when a reference has already been cited:
```
FN 1: John Doe, A book, Publisher, Year.
FN 25: Doe (see FN 1). 
```

**Comment by adunning** [comment]
Providing full notes with each new chapter is a fairly common requirement for note citations. For example, [MHRA 7.12](https://www.mhra.org.uk/style/chapter7.html):

> In a monograph or thesis […] give a full reference for a source at its first mention in the chapter, and shortened references later in that chapter. If it occurs again in a later chapter, give the full reference again at its first mention there.

Achieving this should be as simple as resetting the subsequent status of citations every time Pandoc encounters a heading of a certain level (e.g. heading 2 or above). This should be a perfect task for a Lua filter, but is this citeproc status accessible?

**Reply by jgm** [reply]
No, the citeproc function just takes a whole document and returns a document with citations.

Using `--file-scope` will work here, but I can't think of any other way to do it.

**Comment by denismaier** [comment]
Maybe something like this: 
- collect all the citations
- split the document into chunks at a certain level
- run citeproc on each of the individual chunks, but don't insert a bibliography. 
- build a bibliography based on the citations that were extracted in the first step

The [recursive-citeproc](https://dialoa.github.io/recursive-citeproc/) filter calls citeproc, so this might serve as inspiration, also [multibib](https://github.com/pandoc-ext/multibib) and [section-bibliographies](https://github.com/pandoc-ext/section-bibliographies). Section bibliographies might actually closest to the what is required here.

Nevertheless, would it be possible to add such a feature natively to pandoc? That would be really useful.

**Reply by adunning** [reply]
I think it can be made even simpler than this: it's only really the `subsequent` status of a citation that needs to be reset at headings of a certain level. I've added a proposal at <https://github.com/jgm/citeproc/issues/160>.
